atlp-rwanda/vikings-ec-fe
# #184821806 User should view items

### What does this PR do?
- This PR implements the way A user can view and interact with items

### Description of Tasks to be completed?
- A user should be able to view all items
- A user  should be able to view single item
- A user can do different operations on the product like wish, add to cart, edit and delete

### How should this be manually tested?
- Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
- Change into the directory vikings-ec-fe
- Checkout to the branch: feature/view-products-184821806
- Run the command npm install
- Run the command npm run dev
- After login you can see products
- You should navigate to `/dashboard/products` to view products as seller

### What are the relevant pivotal tracker stories?
[#184821806](https://www.pivotaltracker.com/story/show/184821806) 

### Screenshots
Implemented products view 
![image](https://user-images.githubusercontent.com/90104592/233777343-149b3b75-547b-46a4-8371-e4dccdc4307e.png)

Implemented single product view 
![image](https://user-images.githubusercontent.com/90104592/233777473-2aad177d-2771-422e-afe1-0ef47e08fbe6.png)

View single product as seller
![image](https://user-images.githubusercontent.com/90104592/233777664-df0c8787-e862-45fe-8129-0088413c30cf.png)
